samples/net/sockets/echo_client/README.rst

```diff
@@ -90,6 +90,36 @@ The certificate and private key used by the sample can be found in the sample's
 :ref:`sockets-echo-server-sample` enable establishing a secure connection
 between the samples.
 
+SOCKS5 proxy support
+====================
+
+It is also possible to connect to the echo-server through a SOCKS5 proxy.
+To enable it, use ``-DOVERLAY_CONFIG=overlay-socks5.conf`` when running ``west
+build`` or  ``cmake``.
+
+By default, to make the testing easier, the proxy is expected to run on the
+same host as the echo-server in Linux host.
+
+To start a proxy server, ``ssh`` can be used.
```

Comment:
Can you reword this to make it clear that ssh has builtin SOCKS server support. E.g. "To start a proxy server, for example a builtin SOCKS server support in ``ssh`` can be used (`-D` option)."